### DISCUSSION TITLE
The Human theme is back!

### DISCUSSION THREAD

**Discussion by AzumaHazuki** [lxqt/lxqt#2425] [Show and tell]
This is another one I've wanted to do for a long time: a port of Ubuntu's oooooold old old Human theme, the one we all remember from Dapper :) This took _ages_ to get right. Included also is an Openbox theme that gets as close to the original Metacity/Gnome 2.x windeco as I could.

![LXQt-Human](https://github.com/lxqt/lxqt/assets/67122280/886235af-6eaa-4a86-b2fb-553b5d870caa)

**Comment by bbbhltz** [comment]
I remember that theme from back when

**Comment by Gnomasz** [comment]
This hit me right in the nostalgia! Dapper was my first installed Linux. This is my theme now. My first impression on the window decorations was "this didn't age well", but it grew on me quickly. And i adore the recolored LXQt logo!
I have paired it with "Human" icon set, though I'm not sure I'll get used to the full color icons in the systray.
